src/test/java/io/mathdojo/useraccountservice/UserAccountServiceApplicationTest.java

```diff
@@ -33,6 +32,8 @@
 import io.mathdojo.useraccountservice.services.SystemService;
 
 @RunWith(SpringRunner.class)
+@ContextConfiguration(classes = TestConfig.class)
+@SuppressWarnings({"rawtypes","unchecked"})
```

Comment:
What does this do?

Reply:
purely cosmetic for compiler warnings. We have a good reason for not adding concrete types to some objects